Translations update from Kodi Weblate

weblate opened the pull request:
Translations update from [Kodi Weblate](https://kodi.weblate.cloud) for [Kodi add-ons: peripherals/peripheral.joystick](https://kodi.weblate.cloud/projects/kodi-add-ons-peripherals/peripheral-joystick/).



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widget/kodi-add-ons-peripherals/peripheral-joystick/horizontal-auto.svg)

Rechi commented on peripheral.joystick/resources/language/resource.language.en_GB/strings.po:
```diff
@@ -0,0 +1,69 @@
+# peripheral.joystick language file
```
This file already exists with lower case `gb`.

garbear replied:
Yup. Translations got fucked. For all add-ons. And weblate is down. Merging this would wreak havoc on all case insensitive filesystems.

weblate committed:
Translated using Weblate (Irish)

Currently translated at 100.0% (11 of 11 strings)

Added translation using Weblate (Hebrew)

Added translation using Weblate (Irish)

Added translation using Weblate (English (United Kingdom))

Added translation using Weblate (Filipino)

Added translation using Weblate (Irish (ga_ie))

Added translation using Weblate (Occidental (ie_GA))

Co-authored-by: Aindriú Mac Giolla Eoin <aindriu80@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Languages add-on <noreply-addon-languages@weblate.org>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-peripherals/peripheral-joystick/ga/
Translation: Kodi add-ons: peripherals/peripheral.joystick

Commit:
Deleted translation using Weblate (English (United Kingdom))

Commit:
Deleted translation using Weblate (Hebrew)

Commit:
Deleted translation using Weblate (Irish)

Commit:
Deleted translation using Weblate (Filipino)

Commit:
Deleted translation using Weblate (Irish (ga_ie))

Commit:
Deleted translation using Weblate (Occidental (ie_ga))

Commit:
Deleted translation using Weblate (Occitan (France) (oc_fr))

Commit:
Deleted translation using Weblate (Kannada (India) (kn_in))

Commit:
Deleted translation using Weblate (Ossetian (os_os))

garbear commented:
Translations should be cleaned up now.
